apps/accounts/templates/provider_registration/preview.html

```diff
@@ -0,0 +1,77 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block head %}
+
+{% endblock %}
+
+{% block content %}
+{{ wizard.form.media }}
+
+<div class="container mx-auto">
+    <form method="post" action="">
+        {% csrf_token %}
+        {{ wizard.management_form }}
+
+        {% comment %}
+        This is a placeholder form - only wizard metadata is being submitted.
+        {% endcomment %}
+
+        <p>Step {{ wizard.steps.step1 }} of {{ wizard.steps.count }}</p>
+
+        <section class="prose my-10">
+            <h1>Your verification request</h1>
+            <p>Confirm that the data is correct and submit your request</p>
+        </section>
+
+        <div>
+            <section class="prose my-10">
+                <h3>About your organisation</h3>
+                {% include "provider_registration/partials/_preview.html" with form=preview_forms.0 %}
+            </section>
+            <section class="prose my-10">
+                <h3>Locations</h3>
+                {% for location_form in preview_forms.1.initial_forms %}
+                {% include "provider_registration/partials/_preview.html" with form=location_form %}
+                {% endfor %}
+            </section>
+            <section class="prose my-10">
+                <h3>Services provided by your organisation</h3>
+                {% include "provider_registration/partials/_preview.html" with form=preview_forms.2 %}
```

Comment:
When I run this locally, I saw all the services offered, regardless of what I added. Can you please check if there is a missing conditional to only show the services a provider is actively offering?

Reply:
Good catch! I reverted the change (https://github.com/thegreenwebfoundation/admin-portal/pull/385/commits/6c226a19f3cf889218456bf658ac75806c03b97a) so that now we display the correct value (only selected services), but as slugs instead of human-readable names. I documented that in the linked Trello card so that we can tackle how to render the values separately from this PR.